### PR TITLE
Stop `prokka` from renaming contigs by default

### PR DESCRIPTION
**Pull request by Darcy220606**
## PR checklist
This PR closes this issue #305 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/funcscan/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/funcscan _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by louperelo — APPROVED** [review]
Looks good to me.
Can this have implications on downstream output names, when tools use the PROKKA annotations? For the summary tools maybe?

**Comment by Darcy220606** [comment]
> Looks good to me. Can this have implications on downstream output names, when tools use the PROKKA annotations? For the summary tools maybe?

Yes, it does @louperelo. Found it out when i was running the benchmarking with prokka. In the summary tables i got different contig names that were not comparable to those when i ran bakta or prodigal. To fix this issue, i added this parametre to retain the contig names. But theres a catch. Prokka gets confused when the contig name is overly long, therefore, this parametre is by default turned off.  By default if annotaion tool is PROKKA, it changes the contig names to PROKKA_#.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.1.4dev - [unreleased]
+
+### `Added`
+
+### `Fixed`
+
+- [###]() Added new parameter `annotation_prokka_retaincontigheaders` to allow prokka to retain the original contig headers/locus tag. (by @darcy220606)
```
Comment (jfy133) [inline CHANGELOG.md:12]: To be updated

**Review by jfy133 — COMMENTED** [review]
I swear we went through loops on this and not doing what we already do broke one of the tools..

But making it optional still works :+1:

File: CHANGELOG.md (hunk 1 above)
Comment (Darcy220606) [reply CHANGELOG.md:12]: exactly :D hence why i added it as optional and not default

**Pull request merged**